MAUI/DataForm/getting-started.md

```diff
@@ -0,0 +1,491 @@
+</td>
+</tr>
+<tr>
+<td>
+{{'[DataFormRadioGroupItem]`Link`
+</td>
+<td>
+Generated for the Enum type property.
+[EnumDataTypeAttribute]
+</td>
+</tr>
+</table>
+
+The following list of editors are supported:
+
+<table>
+<tr>
+<th>Editor</th>
+<th>Data Type/Attribute</th>
+<th>Input control loaded</th>
+</tr>
+<tr>
+<td>
+Text
+</td>
+<td>
+The String type property and any other type apart from the following specified cases.
+</td>
+<td>
+{{'[Entry]`Link`
+</td>
+</tr>
+<tr>
+<td>
+MultilineText
+</td>
+<td>
+The String type property with multi line text.
+[DataType(DataType.Multiline)] 
+</td>
+<td>
+{{'[Editor]`Link`
+</td>
+</tr>
+<tr>
+<td>
+Password
+</td>
+<td>
+The String type property with [DataType(DataType.Password)] attribute.
+</td>
+<td>
+{{'[Entry]`Link`
+</td>
+</tr>
+<tr>
+<td>
+CheckBox
+</td>
+<td>
+Bool type property.
+</td>
+<td>
+{{'[CheckBox]`Link`
+</td>
+</tr>
+<tr>
+<td>
+Switch
+</td>
+<td>
+Bool type property.
+</td>
+<td>
+{{'[Switch] `Link`
+</td>
+</tr>
+<tr>
+<td>
+Picker
+</td>
+<td>
+Enum and List type property. 
+[EnumDataTypeAttribute]
+</td>
+<td>
+{{'[Picker]`Link`
+</td>
+</tr>
+<tr>
+<td>
+DatePicker
+</td>
+<td>
+The DateTime type property with date value.
+[DataType(DataType.Date)]
+[DataType(DataType.DateTime)]
+</td>
+<td>
+{{'[DatePicker]`Link`
+</td>
+</tr>
+<tr>
+<td>
+TimePicker
+</td>
+<td>
+Property with [DataType(DataType.Time)] attribute.
+</td>
+<td>
+{{'[TimePicker]`Link`
+</td>
+</tr>
+<tr>
+<td>
+ComboBox
+</td>
+<td>
+Enum and List type property.
+[EnumDataTypeAttribute]
+</td>
+<td>
+{{'[SfComboBox]`Link`
+</td>
+</tr>
+<tr>
+<td>
+AutoComplete
+</td>
+<td>
+Enum and List type property.
+[EnumDataTypeAttribute]
+</td>
+<td>
+{{'[SfAutoComplete]`Link`
+</td>
+</tr>
+
+<tr>
+<td>
+RadioGroup
+</td>
+<td>
+Enum and List type property.
+[EnumDataTypeAttribute]
+</td>
+<td>
+{{'[VerticalStackLayout]`Link`
+</td>
+</tr>
+</table>
+
+## Layout options
+
+### Label position
+
+By default, the data form arranges the label at left side and input control at the right side. You can change the label position by setting the `SfDataForm.LabelPosition` property. You can position the label from left to top of the input control by setting the `LabelPosition` as Top.
+
+{% tabs %}
+{% highlight MainPage.xaml %}
+
+    <dataForm:SfDataForm x:Name="dataForm" >
+            <dataForm:SfDataForm.DefaultLayoutSettings>
+                <dataForm:DataFormDefaultLayoutSettings LabelPosition="Top"/>
+            </dataForm:SfDataForm.DefaultLayoutSettings>
+    </dataForm:SfDataForm>
+
+{% endhighlight %}
+{% highlight MainPage.xaml.cs %}
+
+    dataForm.DefaultLayoutSettings.LabelPosition = LabelPosition.Top;
+
+{% endhighlight %}
+{% endtabs %}
+
+`Label Positioning images`
+
+### Grid layout
+
+By default, the data form arranges one data field per row. It is possible to have more than one data field per row by setting the `ColumnCount` property which provides grid like layout for the data form.
```

Comment:
Need to mention about ItemsOrderInRow property here.

Reply:
Definition for ItemsOrderInRow added